debian/copyright

```diff
@@ -0,0 +1,33 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+
+Files: *
+Copyright: © 2024 Contributors <info@litecli.com>
```

Comment:
Similar to the above, this should either list all the people involved if it's short, or some catchall depending on your attribution model. It can span multiple lines e.g.

```
 16 Copyright: © 2004-2009 Canonical Ltd.
 17            © 2005 Gustavo Noronha Silva
 18            © 2006-2007 Sebastian Heinlein
```